### PR TITLE
Automate case:audit log related to guest disk

### PR DESCRIPTION
**Pull request by chunfuwen**
Automate case:audit log related to guest disk
    
 ausearch attach/detach audio log while hot-plug/unplug a disk to guest
    
 Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
cleaning libvirtd logs...
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virtual_disks.audit_log_disk.hotplug.volume_backingstore.start_vm

**Comment by chunfuwen** [comment]
Automate xx-89002

**Comment by chunfuwen** [comment]
some libvirt audit related utils had been aggregated into https://github.com/avocado-framework/avocado-vt/pull/3524. So after https://github.com/avocado-framework/avocado-vt/pull/3524 was merged, some code here need be refactored accordingly.

**Comment by chunfuwen** [comment]
@qiankehan  please take a look

**Review by qiankehan — APPROVED** [review]
LTGM

**Comment by chunfuwen** [comment]
@dzhengfy  please help take a review

File: libvirt/tests/src/virtual_disks/virtual_disks_audit_log_disk.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    except virt_vm.VMStartError as e:
+        if status_error:
+            LOG.debug("VM failed to start as expected."
+                      "Error: %s", str(e))
+        else:
+            test.fail("VM failed to start."
+                      "Error: %s" % str(e))
+    else:
+        check_disk_message_from_audit_log('attach', test)
+        virsh.detach_disk(vm_name, params.get("target_dev"))
```
[the remark this answers is not in the document]
Comment (chunfuwen) [reply libvirt/tests/src/virtual_disks/virtual_disks_audit_log_disk.py]: updated

**Comment by chunfuwen** [comment]
retest result after change:

Command 'virsh list' finished with 0 after 0.032497618s
cleaning libvirtd logs...
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virtual_disks.audit_log_disk.hotplug.file_backing.start_vm

**Review by dzhengfy — APPROVED** [review]

**Pull request merged**